Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -169,13 +172,21 @@ public static async Task<(int totalCount, int newCount)> ImportAccountAsync(Func
         private static async Task<List<ImportItem>> scanAccountsAsync(Func<Account, Task<ApiExtended>> apiExtendedfunc, Account[] accounts, LibraryOptions libraryOptions)
         {
             var tasks = new List<Task<List<ImportItem>>>();
-            foreach (var account in accounts)
+
+            using LogArchiver archiver
+                = Log.Logger.IsDebugEnabled()
+                ? new LogArchiver(System.IO.Path.Combine(Configuration.Instance.LibationFiles, "LibraryScans.zip"))
+                : default;
```

Comment:
I really didn't think this work, but good ol' C# comes through again by making sure I don't shoot myself in the foot!

Reply:
The hell? So it created an idisposible ... null???

Reply:
clr must use IDisposible?.Dispose() internally.